app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupingSettings.java

```diff
@@ -0,0 +1,28 @@
+package com.randomappsinc.studentpicker.grouping;
+
+public class GroupingSettings {
+
+    private int numOfNamesInGroup;
+    private int numOfGroups;
+
+    public GroupingSettings() {
+        this.numOfNamesInGroup = 2;
```

Comment:
Having all these hardcoded values is kind of awkward. Let's move this 2 and 1 into integers.xml. Since GroupingSettings is just a model, it can take in 2 ints in the constructor for initial values. The fragment will fetch the default values (since it has a context) and pass them into the constructor.